nuspec/Forms/CoreContent/MvxFormsViewModel.cs.pp

```diff
@@ -0,0 +1,32 @@
+using System.Threading.Tasks;
+using MvvmCross.Core.ViewModels;
+
+namespace $rootnamespace$.Core.ViewModels
+{
+    public class MvxFormsViewModel : MvxViewModel
```

Comment:
Wouldn't it be better to name this `MainViewModel` for consistency with the other StarterPacks?